pkg/cli/push.go

```diff
@@ -9,13 +9,17 @@ import (
 )
 
 func NewPush(c CommandContext) *cobra.Command {
-	return cli.Command(&Push{client: c.ClientFactory}, cobra.Command{
+	cmd := cli.Command(&Push{client: c.ClientFactory}, cobra.Command{
 		Use:               "push [flags] IMAGE",
 		SilenceUsage:      true,
 		Short:             "Push an image to a remote registry",
 		Args:              cobra.ExactArgs(1),
 		ValidArgsFunction: newCompletion(c.ClientFactory, imagesCompletion(false)).withShouldCompleteOptions(onlyNumArgs(1)).complete,
 	})
+	cmd.Flags().Lookup("sign").Hidden = true
+	cmd.Flags().Lookup("key").Hidden = true
+	cmd.Flags().Lookup("signature-annotation").Hidden = true
```

Comment:
nit: I believe you can add `hidden:"true"` to the struct tags below and it will do the same thing.

Reply:
You're right, it does. I like this better so I'm updating it. Thanks!